tests/lib/app_version/git_tweak/VERSION

```diff
@@ -0,0 +1,5 @@
+VERSION_MAJOR = 2
+VERSION_MINOR = 3
+PATCHLEVEL = 4
+VERSION_TWEAK = 0
```

Comment:
This test is an odd one because we don't have a way to determine what value VERSION_TWEAK should end up with inside the test itself, as it will change with the git commit. Instead need to test the values it is not.

I don't care whether it is 0 because it was explicitly set to 0 here, or whether it was set to 0 because something was undefined, I only care that the final value is NOT 0.